src/analysis/asymptotics.lean

```diff
@@ -755,13 +755,13 @@ end
 end
 
 section
-variables [normed_field β]
+variables {K : Type*} [normed_field K]
```

Comment:
I'm completely fine with this, but thought that we had settled on `\bk` in the analysis library.

Reply:
The notation `K` was used in this file, so I used it to be coherent. But you are right that it is better with `\bk`, so I changed the whole file (and linted it).